Editor/Analytics/AnalyticsUtils.cs

```diff
@@ -43,19 +49,13 @@ internal static void SetTrackingTo(AnalyticsState state)
                     // Create new session id, which allows better tracking
                     EditorPrefs.SetString(BaseAnalyticsTracker.KeySessionId, Guid.NewGuid().ToString());
                 }
-
-                EditorPrefExtensions.SetEnum(KeyTrackingState, AnalyticsState.Enabled);
-                SendTrackingEvent(AnalyticsState.Enabled);
             }
-            else if (state == AnalyticsState.Disabled && currentState != AnalyticsState.Disabled)
+
+            if (currentState != state)
             {
-                SendTrackingEvent(AnalyticsState.Disabled);
+                SendTrackingEvent(state);
                 EditorPrefExtensions.SetEnum(KeyTrackingState, AnalyticsState.Disabled);
             }
```

Comment:
This does not work if Tracking was disabled or unknown and is set to enabled, because SendTrackingEvent won't send anything when it is currently not enabled. That's the reason why it was like it was before.

Reply:
Question: is it important to send "I'm now sending data" when we start sending data anyway?

Reply:
Yeah, maybe not that important unless you want to know if someone disables and enables it on the same day. :)